src/Helpers/index.js

```diff
@@ -8,16 +8,53 @@ import mapboxgl from 'Lib';
  */
 const CIRCLE_POINTS_CONFIG = 64;
 
+/**
+ * Array of possible units
+ * @type {Array}
+ */
+export const UNITS = ['kilometers', 'meters', 'miles', 'feet'];
+
+/**
+ * Convert the radius into the unit given
+ * @param {Number} radius unit
+ * @param {String} unit to convert to if necessary
+ */
+export function convertRadiusUnit(radius, unit = 'kilometers') {
+    let convertedRadius = Number(radius);
+    let convertedUnit = unit;
+
+    if (isNaN(radius)) {
+        global.console.error('The radius given is not a number');
+    }
+
+    if (UNITS.indexOf(unit) === -1) {
+        convertedUnit = 'kilometers';
+        global.console.warn(`The unit is not supported, the fallback "${convertedUnit}" is used`);
```

Comment:
You can mention the wrong unit in warning, for better understanding.

`The unit '${unit}' is not supported, fallback to '${convertedUnit}`